gensim/models/word2vec.py

```diff
@@ -424,7 +424,7 @@ class Word2Vec(BaseWordEmbeddingsModel):
 
     def __init__(self, sentences=None, size=100, alpha=0.025, window=5, min_count=5,
                  max_vocab_size=None, sample=1e-3, seed=1, workers=3, min_alpha=0.0001,
-                 sg=0, hs=0, negative=5, cbow_mean=1, hashfxn=hash, iter=5, null_word=0,
+                 sg=0, hs=0, negative=5, num_bits=0, cbow_mean=1, hashfxn=hash, iter=5, null_word=0,
```

Comment:
this is a good idea to have `assert` here (because this works only with CBOW and negative sampling) + num_bits accept 1 and 2 (not more)